eng/pipelines/templates/jobs/ci.yml

```diff
@@ -37,6 +37,15 @@ parameters:
   - name: MavenVerifyAdditionalOptions
     type: string
     default: ''
+  - name: MavenBuildPackageParallelization
+    type: string
+    default: '1C'
+  - name: MavenBuildInstallParallelization
+    type: string
+    default: '2C'
+  - name: SparkShading
+    type: string
+    default: ''
```

Comment:
what is the default value if not provided ?

Reply:
Two lines below each new Parameter :-)